src/rpc/mining.cpp

```diff
@@ -874,7 +874,7 @@ static UniValue getblocktemplate(const JSONRPCRequest& request)
     return result;
 }
 
-class submitblock_StateCatcher : public CValidationInterface
+class submitblock_StateCatcher final : public CValidationInterface
```

Comment:
This is also unrelated cleanup?

Reply:
I don't know, the compiler told me to do it IIRC